### ISSUE TITLE
Reloading gists when switching from gists to local

### ISSUE THREAD

**Issue by TheFern2** [lauthieb/code-notes#96]
**Describe the bug**
Repetitive loading have you waiting on gists loading.

**To Reproduce**
1. Go to Gists, see the loading circle
2. Click on local
3. Go back to gists, and a new loading circle appears

**Expected behavior**
I think the loaded gists should remain loaded even when going to local, and maybe have a refresh button if you want to pull new gists from profile. I have about 20+ gists not much, and I have to wait a few seconds before seeing my gists load.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version1.2.2

**Comment by lauthieb** [comment]
Hello @kodaman2,
Yes you're right, I think it could be nice to have a "refresh" button.

**Comment by kodaman2** [comment]
I'll see what I can come up for this issue.

**Comment by jodli** [comment]
Maybe just use this [package](https://www.npmjs.com/package/vuex-cache)?

**Comment by jodli** [comment]
I've tried it and it seems to work quite nicely. Maybe this is already good enough for now and something to work with?

**Comment by kodaman2** [comment]
I'll take a look at it tonight. Looks like is heading in the right direction. 😀 I still think the reloading needs to be handled another way like maybe Note.js new state property gistsLoaded, and a refresh button tied to the load action and/or a shortcut such as cmd/ctrl + r.

**Comment by jodli** [comment]
You give it a go then. Didn't want to steal this issue from you! :(

**Comment by kodaman2** [comment]
@jodli You're good, I'll try to submit a commit to your PR and that way is both our idea. 👍😁

**Comment by kodaman2** [comment]
Ok, now that I am done with the ghe auth PR, I can get back to this one. It might take me a little longer but cache or avoiding full notes reload will be done.
